docs/conf.py

```diff
@@ -151,9 +168,13 @@ def check_if_v046():
 }
 
 html_sidebars = {
-    "**": ["version-sidebar.html", "sidebar-search-bs.html", "sidebar-nav-bs.html"],
```

Comment:
Any particular reason why these had to be removed?

Reply:
Yeah, I was going to ask you about this too. How could you still build the docs while these files don't exist? It would throw a `TemplateNotFound` error. Or I was missing something here 🤔

Reply:
Turns out `"sidebar-search-bs.html"` has been replaced by `"search-field.html"` according to https://github.com/executablebooks/sphinx-book-theme/issues/329
I've pushed that update